src/components/SkillCardScrollList/SkillCard.js

```diff
@@ -60,36 +51,37 @@ class SkillCard extends Component {
         break;
       default:
         scrollCards = 1;
+        this.setState({ rightBtnDisplay: 'inline' });
     }
-    let width = window.innerWidth - 317;
+    // if(this.state.leftBtnDisplay==='none'&&this.state.rightBtnDisplay==='none'){
```

Comment:
Remove comments. As you are not using this code.

Reply:
ok sir